v1alpha1/proto/grafeas.proto

```diff
@@ -1250,6 +1285,29 @@ message ProjectRepoId {
 // there would be one `Note` for the vulnerability and an `Occurrence` for
 // each package with the vulnerability referencing that `Note`.
 service Grafeas {
+  // Creates a new `Project`.
+  rpc CreateProject(CreateProjectRequest) returns (google.protobuf.Empty) {
```

Comment:
I'd prefer the Project CRUD in a separate service, as I can imagine people having a preexisting project or account structure in place, which may be outside of this.

Reply:
Ok. And then we make the location of the project CRUD service becomes a configuration parameter? 
Name of service? Projects? GrafeasProjects?

Reply:
Yes on project CRUD being a config parameter. 

GrafeasProjects sgtm, it avoids namespace duplication.

Reply:
I've put GrafeasProjects in its own service which I inject into the Grafeas service. Was this about what you had in mind?
It's not finished though.
1. For now I'm using the same storage object for Grafeas and GrafeasProjects. If they should be separated I think the Storager interface should be split as well.
2. The Grafeas service still invokes the GrafeasProject methods using a local object rather than a gRPC client. Not sure gRPC always should be used or if we only call out when projects are located somewhere else. Thoughts?
3. Configuration of GrafeasProjects service location still remains (as a consequence of 2)
4. I realized that I hade put the "make sure project exist" check in the wrong layer before. I had put it in the storage layer. Now it is in the application layer instead (impl.go).

Reply:
GrafeasProjects looks good. 

1. I pondered this when reviewing, I think it makes sense to split Storager, its easy to implement both interfaces in the same package if one wants to - however I can also imagine calling out to a different backend for project info.
2. I think for now we can keep it simple, and use the local object. If we need to split it up later we can. We're running both in the same service now so that's fine.
3. GrafeasProjects can just be added to the existing server. If we want to break it up later we can, but no need to over engineer that now :)
4. Yeah - that looks good.

Reply:
Ok. I'll split the interface. Makes sense to split projects from the rest all the way down.